docs/api-docs/getting-started/authentication.md

```diff
@@ -1,30 +1,469 @@
 # Authentication
 
+BigCommerce offers a suite of APIs that let you manage store data, sign customers in, make client-side queries for product information, and write apps that integrate third-party services into store operations.
 
+This article provides an overview of the authentication schemes that our APIs use organized by the header or other mechanism they use to authenticate with our servers.
 
-BigCommerce has five different APIs that let you manage store data, log in customers, make client-side queries for product information, and more. Each requires a different authentication method.
+Regardless of authentication scheme, BigCommerce **API accounts** play a role in every authenticated request to our servers.
 
-## REST APIs
+Our API accounts come in a few different flavors to meet the needs of different use cases. For example, app API accounts work well in multi-store contexts, whereas store API accounts are a good choice for front-end applications. **Some endpoints only work with one kind of API account.** For a thorough explanation of the differences, check out the section on [choosing the right kind of API account](/api-docs/getting-started/authentication/rest-api-authentication#choosing-the-right-kind-of-api-account) in our API accounts article.
 
-Requests to BigCommerce's V2 and V3 REST APIs require you to pass an access token in the header. For instructions on generating this credential, see [Obtaining Store API Credentials](/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials#obtaining-store-api-credentials).
+## Stable tokens
 
-## Storefront API
-The Storefront API is unauthenticated, allowing you to make client-side requests for carts, checkouts, and orders using JavaScript. 
+Most of our APIs use credentials that do not expire based on a time frame. Depending on the type of API account the credentials belong to, they may expire based on user or developer actions. 
+<!-- For more on events that expire tokens, see ??forthcoming apps article??. -->
 
+The following sections describe the two authentication schemes we currently have that use stable tokens. They include example requests and indicate the endpoints to which the schemes apply.

```

Comment:
they don't actually show. the `access tokens` section describes, and the other one lists.

changed to

```suggestion
The following sections describe the two authentication schemes we use that rely on stable tokens. They include example requests and list or describe the endpoints to which the schemes apply.

```

I'm gonna resolve this because it's one word. feel free to reopen if the solution rankles you!